packages/compass-schema-validation/src/modules/validation.ts

```diff
@@ -484,3 +485,13 @@ export const activateValidation = (): SchemaValidationThunkAction<void> => {
     void dispatch(fetchValidation(namespace));
   };
 };
+
+export const hasErrorAndLogValidationActionSupport = (
+  serverVersion: string
+) => {
+  try {
+    return semver.gte(serverVersion, '8.1.0-rc.0');
```

Comment:
seems worthwhile to do this if someone is trying the RC version for whatever reason. not sure if we'd rather make it stricter and let it just be `8.1.0` as semver is strict about pre-releases otherwise; alternatively we can also `semver.coerce(serverVersion)`